### PR TITLE
First iteration of Message Review display optimizations for small screens

### PR DESCRIPTION
**Pull request by lperson**

### CODE REVIEW

**Comment by joemcl** [comment]
Hey @lperson just merged this successfully but get this error message on deploy and build on Heroku: "Unable to fetch source from: https://codeload.github.com/WorkingFamilies/Spoke/legacy.tar.gz/001728e2fd6d08ec97290ee7422a89d7cb6d1acc"

**Comment by joemcl** [comment]
Hey @lperson so with the bug fix above - 472558a - this is ready for testing?

**Comment by joemcl** [comment]
@lperson also see https://github.com/MoveOnOrg/Spoke/issues/748

File: READEM2.MD [hunk 1]
```diff
@@ -0,0 +1 @@
+dummy file for creating a new empty branch for Staging Spoke instance of MoveOn/Spoke latest branch main
```
Comment (schuyler1d) [inline READEM2.MD]: please remove?  or what's this for?

File: readmestaging20180812 [hunk 2]
```diff
@@ -0,0 +1 @@
+Creating branch staging20180812
```
Comment (schuyler1d) [inline readmestaging20180812]: cleanup

File: readmestaging20180813 [hunk 3]
```diff
@@ -0,0 +1 @@
+Creating branch staging20180813 for deploying to spokewfstagingdeploy
```
Comment (schuyler1d) [inline readmestaging20180813]: cleanup

File: readmestaging20180818 [hunk 4]
```diff
@@ -0,0 +1 @@
+Creating new branch staging20180818
```
Comment (schuyler1d) [inline readmestaging20180818]: cleanup

File: package.json [hunk 5]
```diff
@@ -42,7 +42,8 @@
     "dev": "nf start -w --procfile ./dev-tools/Procfile.dev",
     "dev-nowrap": "nf start --trim 500 --procfile ./dev-tools/Procfile.dev --env ./__test__/e2e/.env.e2e",
     "debug-server": "nf start -w --procfile ./dev-tools/Procfile-debug-server.dev",
-    "dev-other": "nf start -w --procfile ./dev-tools/Procfile-other.dev"
+    "dev-other": "nf start -w --procfile ./dev-tools/Procfile-other.dev",
```
Comment (schuyler1d) [inline package.json]: other.dev doesn't seem to exist, so please remove.

File: src/workers/email-report.js [hunk 6]
```diff
@@ -0,0 +1,183 @@
...
+  output += strings[values.length]
+
+  // Split on newlines.
+  const lines = output.split(/(?:\r\n|\n|\r)/)
+
+  // Rip out the leading whitespace.
+  return lines.map(line => line.replace(/^\s+/gm, '')).join(' ').trim()
+}
+
+(async () => {
```
Comment (schuyler1d) [inline src/workers/email-report.js]: please make this file a library, and then enable running it with a separate file.
Then please add a method in workers/job-processes.js that runs it (see e.g. the run migrations function there) which can make it automatable the same way the other job processes are.

File: src/server/mail.js [hunk 7]
```diff
@@ -45,7 +46,8 @@ export const sendEmail = async ({ to, subject, text, replyTo }) => {
     from: process.env.EMAIL_FROM,
     to,
     subject,
-    text
+    text,
+    html
```
Comment (schuyler1d) [inline src/server/mail.js]: does this still work if html is empty?

File: src/server/api/schema.js [hunk 8]
```diff
@@ -578,7 +559,7 @@ const rootMutations = {
       let createCannedResponses = r
         .knex('canned_response')
         .where({ campaign_id: oldCampaignId })
-        .then(function (res) {
+        .then(function(res) {
```
Comment (schuyler1d) [inline src/server/api/schema.js]: if you're changing white space might as well fix it the way lint would ask and use `(res) => {`...

**Review by schuyler1d — DISMISSED** [review]
this looks good.  Can you cleanup the extra files and then make the workers/email-report.js librarified.  Then I'm :+1: 
Awesome work!

File: readmestaging20180818 (hunk 4 above)
Comment (lperson) [reply readmestaging20180818]: oh yikes!

File: READEM2.MD (hunk 1 above)
Comment (lperson) [reply READEM2.MD]: The WFP folks added this file.  I'll remove it.

Comment (lperson) [reply READEM2.MD]: removed

Comment (lperson) [reply READEM2.MD]: removed -- it got in here because I merged WFP's staging into my branch

File: readmestaging20180812 (hunk 2 above)
Comment (lperson) [reply readmestaging20180812]: removed -- it got in here because I merged WFP's staging into my branch

File: readmestaging20180813 (hunk 3 above)
Comment (lperson) [reply readmestaging20180813]: removed -- it got in here because I merged WFP's staging into my branch

File: readmestaging20180818 (hunk 4 above)
Comment (lperson) [reply readmestaging20180818]: removed -- it got in here because I merged WFP's staging into my branch

File: package.json (hunk 5 above)
Comment (lperson) [reply package.json]: it's not `other.dev`, it's `Procfile-other.dev` and it's definitely in the repository

**Comment by shakalee14** [comment]
@lperson - can you investigate why these tests are failing?

**Comment by lperson** [comment]
@shakalee14 the tests were failing because of a bad merge

**Comment by lperson** [comment]
@shakalee14 thanks for following up

File: src/components/Navigation.jsx [hunk 9]
```diff
@@ -4,39 +4,79 @@ import Paper from 'material-ui/Paper'
...
+            <FlatButton
+              label={'Close Menu'}
+              onClick={props.onToggleMenu}
+            />
+          </div>
+
+          <List>
+            {sections.map(section => (
+              <ListItem
+                {...dataTest(camelCase(`nav ${section.path}`))}
```
Comment (azuzunaga) [inline src/components/Navigation.jsx:47]: I believe `lodash` needs to be imported and `camelCase` needs to be invoked from the import for this to work. Or import `camelCase` from '../lib/attributes'.

**Comment by schuyler1d** [comment]
closing as it's included in #944

**Pull request closed without merging**